Tests/BootCamp.Chapter.Tests/GameComponents/HandTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using BootCamp.Chapter.Gambling;
+using Moq;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests.GameComponents
+{
+    public class HandTests
+    {
+        private Card[] _someCards = new[]
```

Comment:
if you need it to be a list, then make it a list instead of array. No point in converting later.